taskcluster/kinds/docker-image/kind.yml

```diff
@@ -22,21 +22,75 @@ tasks:
         args:
             PYTHON_VERSION: "3.8"
             PYTHON_REQ_SUFFIX: ".py38"
-    pushapkscript-python38:
-        definition: pushapkscript
-        parent: python38
-    pushapkscript-python39:
-        definition: pushapkscript
-        parent: python39
-    pushflatpakscript-python38:
-        definition: pushflatpakscript
-        parent: python38
-    pushflatpakscript-python39:
-        definition: pushflatpakscript
-        parent: python39
-    signingscript-python38:
-        definition: signingscript
-        parent: python38
-    signingscript-python39:
-        definition: signingscript
-        parent: python39
+
+    # Used by push-image kind
+    skopeo: {}
+
+    base:
+        args:
+            BASE_IMAGE: python:3.9.7
+    # TODO: Figure out PYTHON_REQ_SUFFIX when we decide to have py3.10
+    # base-3.10:
+    #     definition: base
+    #     args:
+    #         BASE_IMAGE: python:3.10
+    addonscript:
+        definition: script
+        parent: base
+        args:
+            SCRIPT_NAME: addonscript
+    balrogscript:
+        definition: script
+        parent: base
+        args:
+            SCRIPT_NAME: balrogscript
+    beetmoverscript:
+        definition: script
+        parent: base
+        args:
+            SCRIPT_NAME: beetmoverscript
+    bouncerscript:
+        definition: script
+        parent: base
+        args:
+            SCRIPT_NAME: bouncerscript
+    githubscript:
+        definition: script
+        parent: base
+        args:
+            SCRIPT_NAME: githubscript
+    shipitscript:
+        definition: script
+        parent: base
+        args:
+            SCRIPT_NAME: shipitscript
+    pushapkscript:
+        parent: base
+    pushflatpakscript:
+        parent: base
+    pushmsixscript:
+        parent: base
+    signingscript:
+        parent: base
+    treescript:
+        parent: base
```

Comment:
Would it be possible to merge the `base` and `script` Dockerfiles? It looks like `script` is just installing the `base.txt` requirements, isn't that needed for these ones with a custom image anyway?

Reply:
The dockerfile for some of the scripts need a lot of changes+testing before it can use the generic + script installed. I feel like that should come from separate PRs